packages/legacy/core/App/components/inputs/PINInput.tsx

```diff
@@ -23,13 +23,17 @@ const PINInput: React.FC<PINInputProps & React.RefAttributes<TextInput>> = forwa
     // const accessible = accessibilityLabel && accessibilityLabel !== '' ? true : false
     const [PIN, setPIN] = useState('')
     const [showPIN, setShowPIN] = useState(false)
-    const [props, getCellOnLayoutHandler] = useClearByFocusCell({
-      value: PIN,
-      setValue: setPIN,
-    })
     const { t } = useTranslation()
     const { TextTheme, PINInputTheme } = useTheme()
     const cellHeight = 48
+    const myOnChangeText = (value: string) => {
```

Comment:
Looks good but can we name this function `onChangeText` and pass it directly to the `CodeField` prop of the same name?

Reply:
done